pages/docs/tutorials/native/mapping-primitive-data-types-from-c.md

```diff
@@ -0,0 +1,116 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Mapping Primitive Data Types from C"
+description: "Primitive Data types from C and how they look in Kotlin/Native side"
+authors: Eugene Petrenko 
+date: 2018-07-11
+showAuthorInfo: false
+issue: EVAN-5343
+---
+
+In this tutorial we learn how C data types are visible in Kotlin/Native and vice versa. You will: 
+- See what [Data Types are in C Language](#types-in-c-language)
+- Create a [tiny C Library](#example-c-library) that uses those types in exports
+- [Inspect Generated Kotlin/Native APIs from a C library](#inspecting-generated-kotlinnative-apis-for-a-c-library)
+- Find how [Primitive Types in Kotlin/Native](#primitive-types-in-kotlinnative) are mapped to C
+
+## Types in C Language
+
+What types do we have in C language? Let's first all of them. I use the
```

Comment:
"Let's first all of them"? ->
Maybe "There are following types in C programming language:"

Reply:
Typoe, fixed, improved